internal/controller/vrg_volrep.go

```diff
@@ -1611,35 +1624,55 @@ func (v *VRGInstance) checkResyncCompletionAsSecondary(pvc *corev1.PersistentVol
 	return true
 }
 
-// isVRConditionMet returns true if the condition is met, and an error mesage if we could not get the
-// condition value.
+type conditionState string
+
+const (
+	conditionMissing = conditionState("missing")
+	conditionStale   = conditionState("stale")
+	conditionUnknown = conditionState("unknown")
+	conditionKnown   = conditionState("known")
```

Comment:
while reviewing, I got confused mixing `conditionKnown` and `conditionUnknown`. I think it is better to rename `conditionKnown` to `conditionExists`

Reply:
conditionExists is confusing with conditionMissing, but I can find a term that different from Unknown.